lldb/source/Plugins/OperatingSystem/Swift/OperatingSystemSwift.cpp

```diff
@@ -0,0 +1,164 @@
+//===-- OperatingSystemSwift.cpp -----------------------------------------===//
```

Comment:
Not important. "OperatingSystemSwift" sounds slightly wrong to me. What about "OperatingSystemSwiftTasks" or "OperatingSystemSwiftConcurrency"?

Reply:
Yeah the name seems misleading the full lldb affordance name is "Operating System Thread Plugin"

Reply:
> the full lldb affordance name is "Operating System Thread Plugin"

While this may be true, it's worth noting that the only other plugin also uses the short version: "OperatingSystemX", where  X = Python.


> OperatingSystemSwiftTasks

I'm going to rename to this!